integration_test/ops_agent_test/main_test.go

```diff
@@ -4650,9 +4651,13 @@ traces:
 			"workload.googleapis.com/WORKLOAD.GOOGLEAPIS.COM/otlp.test.prefix4",
 			"workload.googleapis.com/WORKLOAD.googleapis.com/otlp.test.prefix5",
 		} {
-			if _, err = gce.WaitForMetric(ctx, logger, vm, name, time.Hour, nil, false); err != nil {
+			ts, err := gce.WaitForMetric(ctx, logger, vm, name, time.Hour, nil, false)
+			if err != nil {
 				t.Error(err)
 			}
+			if _, ok := ts.Metric.Labels["service_name"]; !ok {
```

Comment:
We may as well check all of the service attributes here, especially since we already know it's handled somewhat manually in the exporter. We can comment out the `service.version` check until [this](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/1065) is merged.